week2/homework/src/todo.js

```diff
@@ -0,0 +1,76 @@
+const fs = require('fs');
+
+const loadJSON = () => {
+  try {
+    const dataBuffer = fs.readFileSync('todos.json');
+    const dataJSON = dataBuffer.toString();
+    return JSON.parse(dataJSON);
+  } catch (e) {
+    return [];
+  }
+};
+
+const saveJSON = todos => {
+  const dataJSON = JSON.stringify(todos);
+  fs.writeFileSync('todos.json', dataJSON);
+};
+
+const createOrderValue = array => {
+  array.forEach((elem, index) => {
+    return (elem.order = index + 1);
+  });
+};
+
+const addToDo = (title, order) => {
+  const todo = loadJSON();
+
+  todo.push({ title, order });
+  createOrderValue(todo);
+  console.log('New to-do added');
+
+  saveJSON(todo);
+};
+
+const removeToDo = order => {
+  const todo = loadJSON();
+
+  const keepToDo = todo.filter(elem => elem.order !== order);

```

Comment:
Your code is correct but it might be simpler to write 
`todo.splice(order-1, 1); ` removes one element at position order-1 from the array

Reply:
cool Thank you!